code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -294,7 +294,7 @@
 			if(mood.sanity <= SANITY_DISTURBED)
 				msg += "[t_He] seem[p_s()] distressed.\n"
 				SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "empath", /datum/mood_event/sad_empath, src)
-			if(mood.mood >= 5) //So roundstart people aren't all "happy"
+			if(mood.sanity >= SANITY_GREAT) //So roundstart people aren't all "happy"

```

Comment:
This doesn't prevent the meta-power of guessing who is a baddie just by looking into thir soul, it just raises the cap and makes it a little less istant and thus not happen at roundstart, one could still potentially guess what's up afterward.
You should think up of a better band-aid, like making a a var to keep track of some "fake" sanity level or just remove this altogheter.